### PR TITLE
Ensure the pathname of the rootUrl is used in the mobile URL

### PR DESCRIPTION
**Pull request by znewsham**
This has been an annoyance for nearly 3 years - as documented here: https://github.com/meteor/meteor/issues/8883 finally getting around to submitting a PR. If you have a non `/` ROOT_URL, the suffix is stripped off when building for cordova, so your app never connects. E.g., `https://mydomain.com/admin`. 

This package ensures that both the parseUrl function and the cordova `generateBootstrapPage` function respect the suffix of ROOT_URL. I've been making this fix to every version of meteor on every build machine for almost 3 years. Some of my projects use a `/suffix` url and some don't.

### CODE REVIEW

**Comment by znewsham** [comment]
@filipenevola I can only fix it by changing the test specification - as the purpose of the code change is to respect the pathname parameter. Is that OK?

**Comment by znewsham** [comment]
@filipenevola tests passing

File: tools/utils/utils.js [hunk 1]
```diff
@@ -44,11 +44,15 @@ exports.parseUrl = function (str, defaults) {
   // for consistency remove colon at the end of protocol
   parsed.protocol = parsed.protocol.replace(/\:$/, '');
 
-  return {
+  var ret = {
```
Comment (filipenevola) [inline tools/utils/utils.js:47]: We should use `let` and not `var`

Comment (znewsham) [reply tools/utils/utils.js:47]: @filipenevola I'm easy with either - but `var` is used everywhere else in this file. I kept it the same for consistency.

**Review by filipenevola — APPROVED** [review]

**Pull request merged**